atlas/schema.hcl

```diff
@@ -1,11 +1,492 @@
+  column "project_id" {
+    null = false
+    type = bigint
+  }
+  column "income_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  primary_key {
+    columns = [column.project_id, column.income_id]
+  }
+  foreign_key "project_id" {
+    columns     = [column.project_id]
+    ref_columns = [table.project_v1.column.project_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+  index "income_id_idx" {
+    unique  = true
+    columns = [column.income_id]
+  }
+}
+
+table "organization_snapshot_v1" {
+  schema = schema.public
+  comment = "A snapshot in time of a given organization entry. A new snapshot is created every time the organization entry is modified."
+  column "organization_id" {
+    null = false
+    type = bigint
+  }
+  column "snapshot_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  column "title" {
+    null = false
+    type = text
+    comment = "The name of organization."
+  }
+  column "income_type" {
+    null = false
+    type = text
+  }
+  column "industry_code" {
+    null = false
+    type = text
+  }
+  column "main_product" {
+    null = true
+    type = text
+    comment = "The main product/service sold by the organization. This field is optional on the T2125 form."
+  }
+  column "partnership" {
+    null = false
+    type = bool
+    default = false
+    comment = "An attribute of the T2125 form - additional info required if the business is a partnership."
+  }
+  column "address" {
+    null = true
+    type = text
+    comment = "The organization's physical address if applicable."
+  }
+  column "create_time" {
+    null = false
+    type = timestamptz
+  }
+  primary_key {
+    columns = [column.organization_id, column.snapshot_id]
+  }
+  foreign_key "organization_id" {
+    columns     = [column.organization_id]
+    ref_columns = [table.organization_v1.column.organization_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+}
+table "organization_v1" {
+  schema = schema.public
+  comment = "Refer to the link for explanations of the organization table fields: https://help.wealthsimple.com/hc/en-ca/articles/4408339655323-How-do-I-report-my-self-employment-income-on-a-T2125."
+  column "organization_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+ column "title" {
+    null = false
+    type = text
+    comment = "The name of organization."
+  }
+  column "income_type" {
+    null = false
+    type = text
+    comment = "This is either Commission or Business/Professional. See link for more information."
+  }
+  column "industry_code" {
+    null = false
+    type = text
+    comment = "The code pertaining to the industry the business is related to. These are mostly used for statistical purposes."
```

Comment:
Perhaps add a link to the spec for industry codes, specifically because it is an industry *code*